Xamarin.PropertyEditing.Mac.Standalone/Xamarin.PropertyEditing.Mac.Standalone.csproj

```diff
@@ -114,4 +114,19 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Mac\Xamarin.Mac.CSharp.targets" />
+  <ItemGroup>
+    <PropertyEditingResource Include="PropertyEditingResource\**\*" />
+  </ItemGroup>
+  <Target Name="AfterBuild">
+    <ItemGroup>
+      <PropertyEditingResourceBundlePath Include="$(SolutionDir)\Xamarin.PropertyEditing.Mac\PropertyEditingResource\**\*" />
+    </ItemGroup>
+    <Copy 
+      SourceFiles="@(PropertyEditingResourceBundlePath)" 
+      DestinationFiles="@(PropertyEditingResourceBundlePath->'$(OutDir)\PropertyEditingResource.bundle\%(RecursiveDir)%(Filename)%(Extension)')" 
+      SkipUnchangedFiles="true" 
+      OverwriteReadOnlyFiles="true" 
+      Retries="3" 
+      RetryDelayMilliseconds="300" />

```

Comment:
If we made this part of `Xamarin.PropertyEditor.Mac.csproj` then it may be easier to ensure the files are copied everywhere they should be :)

Reply:
I tweaked it to replicate what we did in other places in the code, and the end result is this:

https://gist.github.com/alanmcgovern/d86dc42e1314db1f32f04f7cc3306caf

I think that ensures the files are always in the right place